go.mod

```diff
@@ -16,6 +16,7 @@ require (
 
 require (
 	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
```

Comment:
why is this added as indirect all of a sudden? can you check where is this used?

Reply:
think it was added while doing experimenting code changes and missed out from cleanup, will remove it.